Hw5/main.cpp

```diff
@@ -71,11 +68,11 @@ void task3()
 	std::cout << std::format("Your number: {}", factorial(number));
 }
 
-std::string binaryString(std::size_t count)
+std::string stripedString(std::size_t count)
 {
 	std::string res("");
 
-	for (std::size_t i = 0; i < count; ++i)
+	for (std::size_t i{ 0 }; i < count; ++i)
 		res.insert(res.begin(), (i & 1) ? '0' : '1');
```

Comment:
You insert value at the beginning of the string. It's not an effective operation. Try to create better solution

Reply:
Changed.